_drafts/2024-01-09-escalabilidade.md

```diff
@@ -75,43 +75,41 @@ O `Throughput`, de maneira geral, descreve o **número de operações que um sis
+O `Tempo de Resposta` é o **tempo total que leva desde o envio de uma solicitação até o recebimento da resposta**. Ele inclui tanto a latência (tempo de ida e volta da rede) quanto o tempo de processamento no servidor, sendo uma soma de ambos na maioria dos casos. Em um contexto de usuário final, é o tempo que leva desde que um usuário realiza uma ação (como clicar em um link ou pressionar uma tecla) até que ele veja o resultado dessa ação, e portanto é medido entre um client um servidor. Em sistemas escaláveis, é importante que o tempo de resposta não aumente significativamente à medida que a utilização da aplicação se eleva.
 
 A `Latência` pode ser definida como o **atraso de rede ou o tempo que uma solicitação leva para viajar do remetente ao receptor**. Em outras palavras, é o atraso entre o início de uma ação e o início da reação, podendo ser influenciado pela distância física entre os comunicantes, pela velocidade do meio de transmissão e por qualquer atraso introduzido por dispositivos intermediários, como roteadores.
 
 O `Tempo de Processamento` é o tempo que um sistema leva para **processar uma solicitação após recebê-la**. Esse termo é frequentemente usado para descrever o tempo necessário para que uma CPU ou servidor processe uma tarefa específica.
 
-O `Tempo de Resposta` é o **tempo total que leva desde o envio de uma solicitação até o recebimento da resposta**. Ele inclui tanto a latência (tempo de ida e volta da rede) quanto o tempo de processamento no servidor, sendo uma soma de ambos na maioria dos casos. Em um contexto de usuário final, é o tempo que leva desde que um usuário realiza uma ação (como clicar em um link ou pressionar uma tecla) até que ele veja o resultado dessa ação.
+O cálculo do tempo de resposta pode ser representado da seguinte forma, a partir do cliente, subtraindo o tempo de início da requisição do tempo final da resposta observada:
 
-Além disso, o cálculo do tempo de resposta pode ser representado da seguinte forma, a partir do cliente, subtraindo o tempo inicial da requisição do tempo final da resposta observada:
-
-\begin{equation} 
+\begin{equation}
 \text{Tempo de Resposta} = \text{Timestamp da Resposta} - \text{Timestamp da Requisição}
-\end{equation} 
+\end{equation}
 
-Todas as três variáveis apresentadas podem ser observadas e medidas independentemente de acordo com as necessidades do ambiente, inclusive sendo uma boa prática para executar troubleshooting mais granular em investigações de problemas, como identificar em que ponto exato da transação ocorreu a degradação em questão.
+Todas as três variáveis apresentadas podem ser observadas e medidas independentemente e em diversos pontos da operação, de acordo com as necessidades do ambiente, inclusive sendo uma boa prática para executar troubleshooting mais granular em investigações de problemas, como identificar em que ponto exato da transação ocorreu a degradação em questão.
```

Comment:
Eu não entendi muito bem o que esse parágrafo quis dizer. Eu acho que a ideia é dizer que você não precisa  medir só o tempo de ponta-a-ponta, mas pode tomar medidas também ao longo do trajeto para obter métricas mais granulares?

Se esse for o caso, eu adicionei uma frase para tentar deixar isso mais claro, mas se _não_ for o caso pode apagar a sugestão :sweat_smile:, e seria bom dar uma clarificada.